lib/portlayer/storage/vsphere/image.go

```diff
@@ -395,7 +397,12 @@ func (v *ImageStore) scratch(op trace.Operation, storeName string) error {
 
 	// Write the metadata to the datastore
 	metaDataDir := v.imageMetadataDirPath(storeName, portlayer.Scratch.ID)
-	if err := writeMetadata(op, v.ds, metaDataDir, nil); err != nil {
+	scratchMetaData := &metadata.ScratchMetaData{
+		ScratchVersion: feature.ScratchBaseStructureVersion,
+	}
+	meta := make(map[string][]byte)
+	meta["metaData"] = []byte(fmt.Sprintf("%+v", scratchMetaData))
```

Comment:
@hickeng The key is used as the name of the metaData file.  So by setting the key to `metaData`, I generate a file named `metaData` with the content of `{scratch_version: 9}` where `9` is the value of `feature.ScratchBaseStructureVersion`. Are you talking about setting the content of the metaData file to `{portlayer.storage.scratch: 9}`?